### PR TITLE
[IMP] mis_builder: update min/max function signatures

### PR DESCRIPTION
**Pull request by sbidoul**
so min(x, y, ...) and min([x, y, ..]) both work as expected.

### CODE REVIEW

**Comment by laetitia-gangloff** [comment]
:+1:

File: mis_builder/models/mis_builder.py [hunk 1]
```diff
@@ -81,16 +81,16 @@ def _avg(l):
     return sum(l) / float(len(l))
 
 
-def _min(l):
+def _min(*l):
```
Comment (pedrobaeza) [inline mis_builder/models/mis_builder.py]: Why don't call it args for being easily recognisable?

**Comment by lmignon** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by sbidoul** [comment]
This one is good for merging.

**Pull request merged**